packages/core/contracts/proxy-scripts/bot-action-wrappers/PositionSettler.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../../common/implementation/FixedPoint.sol";
+
+// Simple contract used to Settle expired positions using a DSProxy.
+contract PositionSettler {
+    function settle(address financialContractAddress) public returns (FixedPoint.Unsigned memory) {
```

Comment:
Should we also put settleExpired in here? Or do we already have that one in a different script?

Reply:
no you right. it should be settleExpired. I got confused for a second. this script is here to settle EMP positions minted by the liquidator.

Reply:
bump